### PR TITLE
make migrate script work for lw and fix existing migration to not break before users are migrated

### PR DESCRIPTION
**Pull request by b0b3rt**
One of the existing migrations was attempting to operate on `Users` without checking that `Users` was a postgres collection.  Also, the `migrate.js` script had a bunch of hard-coded filenames which didn't match our credentials setup.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1204475100865062) by [Unito](https://www.unito.io)

### CODE REVIEW

**Comment by b0b3rt** [comment]
Would be good to get @oetherington 👀 on this

File: migrate.js [hunk 1]
```diff
@@ -28,12 +28,31 @@ const fetchImports = (args, isProd) => {
...
-const credentialsFile = (fileName) => {
+const credentialsFile = (fileName, forum) => {
   const base = process.env.GITHUB_WORKSPACE ?? "..";
-  return `${base}/ForumCredentials/${fileName}`;
+  const repoName = forum === 'lw' ? 'LessWrong-Credentials/connectionStrings' : 'ForumCredentials';
+  return `${base}/${repoName}/${fileName}`;
 }
 
-const readUrlFile = async (fileName) => (await readFile(credentialsFile(fileName))).toString().trim();
+const settingsFile = (fileName, forum) => {
```
Comment (jpaddison3) [inline migrate.js:37]: `settingsFilePath`?

File: migrate.js [hunk 2]
```diff
@@ -52,6 +71,9 @@ const readUrlFile = async (fileName) => (await readFile(credentialsFile(fileName
     mode = "dev";
   }
 
+  const forum = process.argv[4];
```
Comment (jpaddison3) [inline migrate.js:74]: Please update usage at top of file

File: migrate.js [hunk 3]
```diff
@@ -28,12 +28,31 @@ const fetchImports = (args, isProd) => {
...
+const mongoFileName = (mode, forum) => forum === 'lw' ? `${mode}-db.txt` : `${mode}-db-conn.txt`;
+const postgresFileName = (mode, forum) => forum === 'lw' ? `pg-${mode}-db.txt` : `${mode}-pg-conn.txt`;
+const settingsFileName = (mode, forum) => {
+  if (forum === 'lw') {
+    if (mode === 'prod') {
+      return 'settings-production-lesswrong.json';
+    }
+    return 'settings-local-dev-devdb.json'
+  }
+  return `settings-${mode}.txt`;
```
Comment (jpaddison3) [inline migrate.js:54]:
```suggestion
  return `settings-${mode}.json`;  
```

**Review by jpaddison3 — COMMENTED** [review]
Looks broadly good. I'm mildly suss of the concat line (L90), so I'm not giving you the tick, but if you tag me in slack I'll pop back in here real fast.

I feel basically confident to give this a review, but let me know if you want Ollie, I won't be offending.

**Pull request merged**